source/tests/system/nirfmxbt_session_tests.cpp

```diff
@@ -0,0 +1,117 @@
+#include <gtest/gtest.h>
```

Comment:
FYI: I added a test for two session under one device at Mose's request in the NR changes. It's currently failing but you can add a version of it here if you want for BT. You can see it in [this commit](https://github.com/ni/grpc-device/pull/466/commits/ec2b98fef158208c1280b1bfd5ac42c4e52f8c54), the test name is `TwoInitializedSessionsOnSameDevice_CloseSessions_ClosesDriverSessions`.

I added a Task under my story to fix it. I think its a general issue with RFmx sessions with our core server session management.